xmlserializer/XmlElement.cpp

```diff
@@ -271,3 +262,18 @@ bool CXmlElement::CChildIterator::next(CXmlElement& xmlChildElement)
     return false;
 }
 
+template bool CXmlElement::getAttribute(const std::string &name, std::string &value) const;
+template bool CXmlElement::getAttribute(const std::string &name, uint16_t &value) const;
+template bool CXmlElement::getAttribute(const std::string &name, uint32_t &value) const;
+template bool CXmlElement::getAttribute(const std::string &name, int32_t &value) const;
+template bool CXmlElement::getAttribute(const std::string &name, uint64_t &value) const;
```

Comment:
bool, float, double and string are already covered. most integer types are covered too. Compiling the Parameter Framework does not necessitates more template instances than the one defined here. When this changes, they can be added.